doc/admin.md

```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
+    +----------------------+
+    | Tables_in_murano     |
+    +----------------------+
+    | alembic_version      |
+    | apistats             |
+    | category             |
+    | cf_orgs              |
+    | cf_serv_inst         |
+    | cf_spaces            |
+    | class_definition     |
+    | environment          |
+    | environment-template |
+    | instance_stats       |
+    | locks                |
+    | package              |
+    | package_to_category  |
+    | package_to_tag       |
+    | session              |
+    | status               |
+    | tag                  |
+    | task                 |
+    +----------------------+
+    18 rows in set (0.00 sec)
+
+
+## Diagnosis Procedures
+
+The Diagnosis Procedures are the first steps that a System Administrator
+will take to locate the source of an error in a GE. Once the nature of
+the error is identified with these tests, the system admin will very
+often have to resort to more concrete and specific testing to pinpoint
+the exact point of error and a possible solution. Such specific testing
+is out of the scope of this section.
+
+### Resource availability
+
+The resource availability in the node should be at least 2Gb of RAM and
+8GB of Hard disk in order to prevent enabler’s bad performance in both
+nodes. This means that bellow these thresholds the enabler is likely to
+experience problems or bad performance.
+
+###  Remote Service Access
+
+We have internally two components to connect, the API and the engine (besides the database manager and the
+message bus). After that two internals component, we
+should connect with the the IdM GE. An administrator to verify that such
+links are available will use this information.
+
+The first step is to check that the api is up and running, for
+this purpose we can execute the following curl command, which is a
+simple GET operation:
+
+    root@fiware:~# curl http://$IP:$PORT/v1.0
+
+The variable will be the IP direction in which we have installed the murano service. This request should return the status of the server if it
+is working properly.
+
+
+In order to check the connectivity between Murano and the IdM
+GE, due to it must obtain a valid token and tenant for a user and
+organization with the following curl commands:
+
+    root@fiware:~# curl
+    -d '{"auth": {"tenantName": "<MY_ORG_NAME>",
+    "passwordCredentials":{"username": "<MY_USERNAME>", "password": "<MY_PASS>"}}}'
+    -H "Content-type: application/json" -H "Accept: application/xml"
+    http://<KEYSTONE_HOST>:<KEYSTONE_PORT>/v2.0/tokens
+
+The will be the name of my Organization/Tenant/Project predefined in the
+IdM GE (aka Keystone). The and variables will be the user name and
+password predefined in the IdM GE and finally the and variables will be
+the IP direction and port in which we can find the IdM GE (aka
+Keystone). This request should return one valid token for the user
+credentials together with more information in a xml format:
```

Comment:
Please use application/json, xml is not supported in FIWARE anymore I think and give very bad impression to put xml here.